runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/TabInterface.java

```diff
@@ -283,7 +284,12 @@ public void handleClick(MenuOptionClicked event)
 			return;
 		}
 
-		if (iconToSet != null)
+		if (event.getMenuOption().equals("Search") && activeTab != null && client.getWidget(WidgetInfo.BANK_SEARCH_BUTTON_BACKGROUND).getSpriteId() != SpriteID.EQUIPMENT_SLOT_SELECTED)
```

Comment:
You should probably check for activeTab not being null first as its the cheapest check, also move the different checks to new lines like in the REMOVE_TAG if statement.